hpcflow/sdk/core/parameters.py

```diff
@@ -1598,19 +1599,31 @@ def _values_from_latin_hypercube(
             optimization=optimization,
             rng=rng,
         )
+
+        parameter_ranges = np.array(
+            [bounds[path] if path in bounds else [0, 1] for path in paths]
+        ).T
+
+        lower_bound = parameter_ranges[0]
+        upper_bound = parameter_ranges[1]
+
         try:
             sampler = LatinHypercube(**kwargs)
         except TypeError:
             # `rng` was previously (<1.15.0) `seed`:
             kwargs["seed"] = kwargs.pop("rng")
             sampler = LatinHypercube(**kwargs)
-        return sampler.random(n=num_samples).T
+
+        samples = scale(sampler.random(n=num_samples), lower_bound, upper_bound).T
+
+        return samples
 
     @classmethod
     def from_latin_hypercube(
         cls,
         paths: Sequence[str],
         num_samples: int,
+        bounds: dict[str, Sequence[float]],
```

Comment:
Similarly, add a default value of `None` to `bounds` here, and make it a keyword-only argument.